api/docs.go

```diff
@@ -76,6 +76,18 @@ const docTemplate = `{
                         "description": "Search term for name and url.",
                         "name": "search",
                         "in": "query"
+                    },
+                    {
+                        "type": "string",
+                        "description": "Filter to repository that matches name exactly",
```

Comment:
```suggestion
                        "description": "Filter repository for by name that matches exactly",
```

Reply:
repositories ? More than one is being filtered to find the exact match. 
"Filter repository by name that matches exactly" sounds like the name has to match exactly before you start to filter. How about:
Filter repositories by name using an exact match.